configure.ac

```diff
@@ -308,6 +308,12 @@ AC_ARG_WITH(
 	[with_openssl_engine="auto"]
 )
 
+AC_ARG_WITH(mptcp,
+	[AS_HELP_STRING([--without-mptcp],[Disable Multipath TCP support])],
```

Comment:
> It looks like they prefer to use `--disable-xxx`, instead of `--without-xxx`.
> 
> Please add a space after the comma: `mptcp], [Disable`



> you also need a `@<:@default=yes@:>@` at the end